sources/Inputs/src/Configuration.cpp

```diff
@@ -41,6 +42,28 @@ updateValue(T& value, const boost::property_tree::ptree& tree, const std::string
     value = value_opt->get_value<T>();
   }
 }
+
+/**
+ * @brief Helper function to update the active power compensation parameter
```

Comment:
Please specify here that if the value is incorrect then the value is not updated